SECP-service/src/main/resources/assets/app/js/login/login-controller.js

```diff
@@ -1,47 +1,47 @@
 angular.module('SECP')
     .controller('LoginController', function ($scope, Auth, $location, $webCrypto) {
         $scope.login = function () {
-
-            Auth.login($scope.user).then(function(res) {
-                if(res.status == 200) {
-                    swal('Welcome to SECP!', 'Login successful!','success');
+            // TODO: NEED TO SEND AUTHENTICATION EMAIL (IF REQUIRED BY ADMIN) CONTAINING VERIFICATION CODE WHEN SUCCESSFULLY LOGGED IN
```

Comment:
what does the authentication code do. Like does it help you reset your password?

Reply:
Its the 2 factor authentication. So if an admin sets the setting for 2 factor authentication to be on, then you have to provide your login info, then provide the code that was sent to your email.

Reply:
I'm guessing this would be done ONLY when the user logs in with a new device?

Reply:
That I'm not sure of. Need to discuss this further.

Reply:
That's a good idea, I think the first time the user signs in we can ask them to enter a passcode that we send to their email.